<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtlMovieEditTask</class>
 <widget class="QWidget" name="QtlMovieEditTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>411</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupInputFile">
     <property name="title">
      <string>Input file</string>
     </property>
     <layout class="QGridLayout" name="layoutInput">
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonInputFile">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtlLineEdit" name="editInputFile"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelInputFile">
        <property name="text">
         <string>File :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupVideo">
     <property name="title">
      <string>Video</string>
     </property>
     <layout class="QVBoxLayout" name="layoutVideo">
      <item>
       <widget class="QtlButtonGrid" name="boxVideoStreams" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAudio">
     <property name="title">
      <string>Audio</string>
     </property>
     <layout class="QVBoxLayout" name="layoutAudio">
      <item>
       <widget class="QtlButtonGrid" name="boxAudioStreams" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSubtitles">
     <property name="title">
      <string>Subtitles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QtlButtonGrid" name="boxSubtitleStreams" native="true">
        <widget class="QRadioButton" name="radioButtonNoSubtitle">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>48</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>None</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonSubtitleFile">
        <property name="text">
         <string>File :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtlLineEdit" name="editSubtitleFile"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonSubtitleFile">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOutputFile">
     <property name="title">
      <string>Output file</string>
     </property>
     <layout class="QGridLayout" name="layoutOutput">
      <item row="0" column="1">
       <widget class="QtlLineEdit" name="editOutputFile"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonOutputFile">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelOutputFile">
        <property name="text">
         <string>File :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QtlButtonGrid" name="boxOutputTypes" native="true"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QWidget" name="boxAspectRatio" native="true">
        <layout class="QHBoxLayout" name="layoutForceDAR">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkForceDAR">
           <property name="text">
            <string>Force Display Aspect Ratio :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QtlLineEdit" name="editForceDAR">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="inputMask">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerForceDAR2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="spacerForceDAR3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="spacerForceDAR1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtlLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QtlLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtlButtonGrid</class>
   <extends>QWidget</extends>
   <header>QtlButtonGrid.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>editInputFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>inputFileNameEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editOutputFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>outputFileNameEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonInputFile</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>selectInputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOutputFile</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>selectOutputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSubtitleFile</sender>
   <signal>clicked()</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>selectSubtitleFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNoSubtitle</sender>
   <signal>toggled(bool)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>inputStreamSelectionUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSubtitleFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>inputStreamSelectionUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSubtitleFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>inputStreamSelectionUpdated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkForceDAR</sender>
   <signal>toggled(bool)</signal>
   <receiver>editForceDAR</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkForceDAR</sender>
   <signal>toggled(bool)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>forcedDarChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editForceDAR</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtlMovieEditTask</receiver>
   <slot>forcedDarChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectInputFile()</slot>
  <slot>selectOutputFile()</slot>
  <slot>selectSubtitleFile()</slot>
  <slot>inputFileNameEdited()</slot>
  <slot>inputStreamSelectionUpdated()</slot>
  <slot>outputFileNameEdited()</slot>
  <slot>showInputFileProperties()</slot>
  <slot>showAudioTest()</slot>
  <slot>forcedDarChanged()</slot>
 </slots>
</ui>
